config.default.json

```diff
@@ -67,18 +67,30 @@
           "headers": {
             "x-broker-content-type": "application/x-www-form-urlencoded"
           }
+        },
+        {
+          "url": "https://$SONAR_QUBE_HOST_URL/system/ping",
+          "method": "GET",
+          "auth": {
+            "type": "header",
+            "name": "Authorization",
+            "value": "Bearer $SONAR_QUBE_API_TOKEN"
+          }
         }
       ],
       "default": {
         "CHECKMARX": "$CHECKMARX",
         "CHECKMARX_USERNAME": "$CHECKMARX_USERNAME",
-        "CHECKMARX_PASSWORD": "$CHECKMARX_PASSWORD"
+        "CHECKMARX_PASSWORD": "$CHECKMARX_PASSWORD",
+        "SONAR_QUBE_HOST_URL": "$SONAR_QUBE_HOST_URL",
+        "SONAR_QUBE_API_TOKEN": "$SONAR_QUBE_API_TOKEN"
       },
       "required": {
         "CHECKMARX": "checkmarx.customer.com",
         "CHECKMARX_USERNAME": "<username>",
         "CHECKMARX_PASSWORD": "<password>",
-        "BROKER_CLIENT_URL": "https://<broker.client.hostname>:<port>"
+        "SONAR_QUBE_HOST_URL": "<sonarqube-host-url>",
```

Comment:
q: SONAR_QUBE or SONARQUBE (no space)?
q: does it make sense to align BASE_URL, HOST_URL, etc?

Reply:
The team brought it like that. Let me ask if there is a particular reasoning here.

Reply:
updating